app/models/manageiq/providers/ibm_power_hmc/infra_manager/host.rb

```diff
@@ -1,6 +1,42 @@
 class ManageIQ::Providers::IbmPowerHmc::InfraManager::Host < ::Host
   include ManageIQ::Providers::IbmPowerHmc::InfraManager::MetricsCaptureMixin
 
+  supports :stop
+  supports :shutdown
```

Comment:
Hey @CallisteH it is turning out to be quite the ordeal to convert Host to supports? from validate_*

For the time being I think it will be faster for us to do
```suggestion
  def validate_stop
    {:available => true,  :message => nil}
  end
  
  def validate_shutdown
    {:available => true,  :message => nil}
  end
```

Until we are able to move everything over to supports